src/main/java/org/sonarlint/intellij/core/ConnectedSonarLintFacade.java

```diff
@@ -93,6 +93,11 @@ protected AnalysisResults analyze(Path baseDir, Path workDir, Collection<ClientI
     return engine.getPluginDetails();
   }
 
+  @Override
+  public ConnectedSonarLintEngine getEngine() {
```

Comment:
Ah, I might have added this one also on my side 😉 making a note to be vigilant during rebase/merge.